cost/azure/instances_log_analitics_utilization/README.md

```diff
@@ -1,66 +1,52 @@
-## Azure Instances Utilization from Log Analytics
+# Azure Instances Utilization from Log Analytics
```

Comment:
the readme header should match the new policy name

Reply:
I have updated the Readme header and also updated the root Readme file.